src/main/java/BotManager.java

```diff
@@ -0,0 +1,150 @@
+import java.util.ArrayList;
+
+public class BotManager {
+    private final String name;
+    private ArrayList<Task> tasks;
+
+    BotManager(String name) {
+        this.name = name;
+        this.tasks = new ArrayList<Task>();
+    }
+
+    void greeting() {
```

Comment:
I would change the name of the function to be greet instead of greeting in line with the coding standard.
```suggestion
    void greet() {
```